src/hw9/ArraySizeException.java

```diff
@@ -1,6 +1,6 @@
 package hw9;
 
-public class ArraySizeException extends Throwable{
-    public ArraySizeException ( String s ){
+public class ArraySizeException extends Throwable {
+    public ArraySizeException (String s) {
```

Comment:
Please apply the correct formatting here. You have `public ArraySizeException (String s) {`, but must be `public ArraySizeException(String s) {`.
Please note that next time I'll lower the mark for that without any additional warnings as I've been telling you about this many times already.